test/input/check-nexus.json

```diff
@@ -5,9 +5,9 @@
             "password": "admin123",
             "repository": {
                 "authenticate": "always",
-                "index_url": "http://192.168.1.106:8081/repository/pypi-private/simple",
-                "repository_url": "http://192.168.1.106:8081/repository/pypi-private/"
+                "index_url": "http://localhost:8081/repository/pypi-private/simple",
```

Comment:
also the IP stuff... you can't do that... 

- first its not our test... 
- second its linked to: https://github.com/Pix4D/concourse-pypi-resource/blob/master/test/ci/params.yml so changing it here would required additional changes...

I don't understand how you end up looking at these files... These files are not needed or used for the pytests. For the CI part we have our own ci directory...

Reply:
But how I can run the tests then?
Nexus instance run in another Docker image, exposing the port 8081.

connecting to 192.168.1.106 from `cfplatformeng/concourse-pypi-resource:latest-rc` just doesn't work...

```
WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPConnection object at 0x77d811e9ef50>: Failed to establish a new connection: [Errno 111] Connection refused')': /repository/pypi-private/simple/test-package1/
```

Either it's my config that is wrong (probably), or the readme is a real mess and there is a lot of wrong things there...

Reply:
I really wonder if someome already ran these tests since we forked this repo...

Reply:
Yes, I did many many times... but only pytest ones... I never had to touch these pipelines... We created our own CI...

I don't know if you are missing some of my comments but... you should never run their pipelines configs....